tests/karate/src/test/java/be/constants.feature

```diff
@@ -126,10 +158,30 @@ Feature: Constants
           return "2PLwVvqxMqW5hQJXkFpNCvBI9MZwuN8rf66V1hS-iZU="
         }
       """
+    * def getIsThisProjectFunQuestionId = call createIsThisProjectFunQuestionId
     * def createIsThisProjectFunVoteId =
+      """
+        function(vote){
+          var JsonConverter = Java.type('be.utils.JsonConverter')
+          var String = Java.type('java.lang.String')
+          var jsonConverter = new JsonConverter()
+          var voteConstant = "Vote"
+          var electionId = getValidElectionSetupId
+          var questionId = getIsThisProjectFunQuestionId
+          return jsonConverter.hash(voteConstant.getBytes(), electionId.getBytes(),
+                                     questionId.getBytes(), vote.getBytes())
+        }
+      """
+    * def createIsThisProjectFunVoteIdVoteYes =
+      """
+        function(){
+          return createIsThisProjectFunVoteId("0")
+        }
+      """
+    * def createIsThisProjectFunVoteIdVoteNo =
       """
         function(){
-          return "d60B94lVWm84lBHc9RE5H67oH-Ad3O1WFflK3NSY3Yk="
+          return createIsThisProjectFunVoteId("1")
```

Comment:
I am not particularly up to date with the spec, but what is the 1 supposed to mean ?

Reply:
1 is the index of the corresponding ballot option

Reply:
Here 0 or 1 would correspond for voting yes or no respectively which are the ballot options available